tests/wpt/web-platform-tests/html/rendering/non-replaced-elements/tables/table-width.html

```diff
@@ -0,0 +1,25 @@
+<link rel="match" href="table-width-ref.html">
```

Comment:
Err, wait, nevermind, that's not the primary issue, is it?  The problem is the interaction between the width specification on table and td.

Here's a testcase which should work in servo:
```html
<table width=0>
<tr>
<td>
a b
</td>
</tr>
</table>
```